src/db.rs

```diff
@@ -16,13 +15,16 @@ use crate::sst::SsTableFormat;
 use crate::sst_iter::SstIterator;
 use crate::tablestore::TableStore;
 use crate::types::ValueDeletable;
+use crate::{block::Block, compactor::Compactor};
 use bytes::Bytes;
 use fail_parallel::FailPointRegistry;
 use object_store::path::Path;
 use object_store::ObjectStore;
 use parking_lot::{Mutex, RwLock};
 use tokio::runtime::Handle;
 
+pub type BlockCache = moka::future::Cache<(SsTableId, usize), Arc<Block>>;
```

Comment:
Any reason you put this here instead of in tablestore.rs? IIRC it was there in your older PR.